app/views/devise/passwords/new.html.erb

```diff
@@ -9,8 +10,9 @@
   </div>
 
   <div class="actions">
-    <%= f.submit "Send me reset password instructions" %>
+    <%= f.submit "Send me reset password instructions", class:"this-thing" %>
```

Comment:
agree, i was mad at it because it wouldn't override the foundation css. so I just called it that lol, and it magically worked